api/buckets/service.go

```diff
@@ -1755,7 +1755,7 @@ func (s *Service) sumBytesPinned(ctx context.Context, delta int64) error {
 	}
 	u := userFromContext(ctx)
 	if u == nil {
-		return fmt.Errorf("context must be from an org/dev/user")
```

Comment:
I should have caught this in the limits PR. This error makes the service unusable from `buckd`, which never has hub accounts/users. It's safe to just ignore if there's no account/user, since the hub interceptor won't let us get this far if they are missing.